walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,107 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+
 
 public final class FindMeetingQuery {
+
+  /*
+   Criterion functional interface is used to define who's (mandatory or all attendees')
+   occupied hours should be taken into account in the findAvailableTimeRanges function.
+   */
+  @FunctionalInterface
+  private interface Criterion {
+    boolean satisfies(int x);
+  }
+  private final Criterion COMMON_CRITERION = x -> x == 0;
+  private final Criterion MANDATORY_CRITERION = x -> x <= 1;
+
+  private static final int MINUTES = 60 * 24;
```

Comment:
Consider updating the name to something that reflects the duration that this represents, e.g., `MINUTES_IN_DAY`.